src/_includes/navbar.html

```diff
@@ -1,9 +1,9 @@
 <header class="navbar">
   <ul aria-label="Main navigation">
```

Comment:
The aria label isn't read out by assistive tech since this isn't a landmark, I would suggest wrapping in a navigation element, e.g. `<nav aria-label="Main">`

Reply:
Wrap the whole `<ul></ul>`?

Reply:
Yes similar to how it's done on dxw.com
```
<nav aria-label="Main">
<ul></ul>
</nav>
```